configs/ats4/auxiliary_software_files/packages.yaml

```diff
@@ -1,6 +1,15 @@
 packages:
   all:
     require: target=x86_64
+  papi:
+    externals:
+    - spec: papi@7.0.0.2
+      prefix: /opt/cray/pe/papi/7.0.0.2
```

Comment:
@becker33 @scheibelp Should we be looking for a system install of PAPI, or is it advisable to build PAPI for each system?